util/memory/tracker.go

```diff
@@ -565,4 +589,13 @@ const (
 	LabelForBindCache int = -22
 	// LabelForNonTransactionalDML represents the label of the non-transactional DML
 	LabelForNonTransactionalDML = -23
+	// LabelForAnalyzeMemory represents the label of the memory of each analyze job
+	LabelForAnalyzeMemory int = -23
+	// LabelForAnalyzeSharedMemory represents the label of the global memory of all analyze jobs
+	LabelForAnalyzeSharedMemory int = -24
```

Comment:
Is it better to say `LabelForGlobalAnalyzeMemory`?

Reply:
Updated.